geowrangler/datasets/geofabrik.py

```diff
@@ -1,17 +1,37 @@
 # AUTOGENERATED! DO NOT EDIT! File to edit: notebooks/05_datasets_geofabrik.ipynb (unless otherwise specified).
 
-__all__ = ["list_geofabrik_regions", "download_geofabrik_region"]
+__all__ = [
+    "list_geofabrik_regions",
+    "download_geofabrik_region",
+    "get_osm_download_url",
+    "download_osm_country_data",
+    "OsmDataManager",
+]
 
 
 # Internal Cell
 import os
 import shutil
+import uuid
 from functools import lru_cache
 from pathlib import Path
+from typing import Union
 from urllib.parse import urlparse
+from urllib.request import HTTPError
+from zipfile import ZipFile
 
-# exporti
+import geopandas as gpd
 import requests
+from fastcore.all import patch, urlcheck
+from loguru import logger
+
+from geowrangler import distance_zonal_stats as dzs
+from geowrangler import vector_zonal_stats as vzs
+
+from .utils import make_report_hook, urlretrieve
+
+# Internal Cell
+DEFAULT_CACHE_DIR = "~/.geowrangler"
```

Comment:
Based on the discussion, we should name this `~/.cache/geowrangler`

Reply:
will implement this.